### PR TITLE
docs(process-engine): config property for bpmn stacktrace log level

### PR DESCRIPTION
**Pull request by danielkelemen**
related to CAM-13977

### CODE REVIEW

File: content/update/minor/717-to-718/_index.md [hunk 1]
```diff
@@ -126,3 +127,9 @@ If you have added custom script tags in one of the `index.html` files of the web
 You don't need to worry about whitelisting for scripts you load via our plugin system.
 
 Find the details in the [Content Security Policy]({{< ref "/webapps/shared-options/header-security.md#content-security-policy" >}}) section.
+
+
+# Log level configuration for BPMN stack trace
+
+We've added a new configuration property called `logLevelBpmnStackTrace` to change the default `DEBUG` level of the bpmn stack trace.
+See the [Logging level parameters]({{< ref "/reference/deployment-descriptors/tags/process-engine.md#logLevelBpmnStackTrace" >}}) section for details.
```
Comment (koevskinikola) [inline content/update/minor/717-to-718/_index.md:135]: Should we also describe what the impact is? For example, tell users that the previous default remains and that they don't need to do any adjustments.

**Review by koevskinikola — CHANGES_REQUESTED** [review]
Looks good. I only have two suggestions:
1. Should we reference the property in the logging section here: https://github.com/camunda/camunda-docs-manual/blob/master/content/user-guide/logging.md#logging-categories?
2. Should we expand the update section to explain to users what they need or needn't do for this change?

File: content/update/minor/717-to-718/_index.md (hunk 1 above)
Comment (danielkelemen) [reply content/update/minor/717-to-718/_index.md]: Good points!
I added some extra infos about the impact and what they could do.
I also referenced the new parameters in the logging user-guide.
Let me know what you think!

**Review by koevskinikola — APPROVED** [review]

File: content/update/minor/717-to-718/_index.md (hunk 1 above)
Comment (koevskinikola) [reply content/update/minor/717-to-718/_index.md]: Looks great!

**Pull request merged**